boards/microbit_v2/README.md

```diff
@@ -90,7 +90,7 @@ The kernel memory layout is different if there is no bootloader. Change the `lay
 MEMORY
 {
   # with bootloader
-  # rom (rx)  : ORIGIN = 0x00010000, LENGTH = 192K
+  # rom (rx)  : ORIGIN = 0x00008000, LENGTH = 192K
```

Comment:
This seems like it's going to be outdated again in no time. Do we need to replicate these values here or could it be easier to just point to the commented out lines in `layout.ld` directly, perhaps using a Markdown link?

Reply:
Putting a link would be ok, just that people tend not to follow it. It might make it more obvious this way.

Reply:
Sure, works for me.